README.md

```diff
@@ -2,36 +2,42 @@
 
 pvmigrate allows migrating PVCs between two StorageClasses by creating new PVs, copying over the data, and then changing PVCs to refer to the new PVs.
 
-## examples
+## Examples
 
 To migrate PVs from the 'default' StorageClass to mynewsc:
 ```bash
 pvmigrate --source-sc default --dest-sc mynewsc
 ```
 
-## process
+## Process
 
 In order, it:
 1. Validates that both the source and dest StorageClasses exist
-1. Finds PVs using the source StorageClass
-1. Finds PVCs corresponding to the above PVs
-1. Creates new PVCs for each existing PVC, but using the new StorageClass
-1. Finds all pods mounting the existing PVCs
-1. Finds StatefulSets and Deployments controlling those pods and sets their scale to 0, recording the original scale for later
-1. Waits for all pods mounting the existing PVCs to be removed
-1. Creates a pod mounting both the original and replacement PVC, which then `rsync`s data between the two (and repeats this for each PVC)
-1. Marks all the PVs associated with the original and replacement PVCs as 'retain', so that they will not be deleted when the PVCs are removed
-1. Deletes the original PVCs so that their names are available, and removes the association between their PV and the removed PVC
-1. Deletes the replacement PVCs so that their PVs are available, and removes the association between their PV and the removed PVC
-1. Creates new PVCs with the original names, but associated with the replacement PVs
-1. Resets the scales of the affected StatefulSets and Deployments
-1. Deletes the original PVs
+2. Finds PVs using the source StorageClass
+3. Finds PVCs corresponding to the above PVs
+4. Creates new PVCs for each existing PVC, but using the new StorageClass
+5. For each PVC:
+    * Finds all pods mounting the existing PVC
+    * Finds all StatefulSets and Deployments controlling those pods and adds an annotation with the original scale before setting that scale to 0
+    * Waits for all pods mounting the existing PVC to be removed
+8. For each PVC:
```

Comment:
What about 6 & 7?

Reply:
ugggggh
apparently Goland's markdown autorenumber feature is just smart enough to get me in trouble